common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4747,6 +4749,9 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         new TimeValidator(TimeUnit.SECONDS),
         "Timeout for Running Query in seconds. A nonpositive value means infinite. " +
         "If the query timeout is also set by thrift API call, the smaller one will be taken."),
+    HIVE_QUERY_EXCLUSIVE_LOCK("hive.query.exclusive.lock", "",
```

Comment:
We should not add this to the hive config variables; the config could be in `org.apache.hadoop.hive.conf.Constants` as others that are used to configure jobs, etc.?